test/test_optimization.jl

```diff
@@ -445,8 +445,8 @@ end
 test_quadratic_optimization()
 test_objective_wrapper()
 test_objective_hessians()
-test_kappa_finding()
-test_bad_a_init()
+#test_kappa_finding()
```

Comment:
These tests are probably less important, but could you still put a comment in about why they're commented out?  (I presume it's the 1D optimization, right?)